venues/ICLR.cc/2018/Conference/webfield/conferenceWebfield_alt.js

```diff
@@ -307,10 +311,25 @@ function renderContent(allNotes, submittedNotes, assignedNotePairs, userGroups,
     fadeIn: false
   });
 
-  if (allNotes.length === PAGE_SIZE) {
+  if (notes.length === PAGE_SIZE) {
     Webfield.setupAutoLoading(BLIND_INVITATION, PAGE_SIZE, submissionListOptions);
   }
 
+  // Withdrawn Papers tab
+  var withdrawnListOptions = _.assign({}, paperDisplayOptions, {
+    showTags: false,
+    container: '#withdrawn-papers'
+  });
+
+  if (withdrawnNotes.length) {
+    Webfield.ui.searchResults(
+      withdrawnNotes,
+      _.assign({}, paperDisplayOptions, {container: '#withdrawn-papers'})
```

Comment:
You can replace this line with the withdrawnListOptions object you created above.